paper/paper.md

```diff
@@ -70,20 +70,17 @@ and refer to \autoref{eq:fourier} from text.
 
 # Summary
 
-`GeophysicalFlows.jl` is Julia package that gathers together a collections of pseudospectral 
-solvers for geophysical fluid dynamics-related partial differential equations on periodic 
-domains. All modules use Fourier-based pseudospectral numerical methods and leverage the 
+`GeophysicalFlows.jl` is a Julia package that contains partial differential solvers for a collection of large-scale geophysical fluid systems on periodic domains. All modules use Fourier-based pseudospectral numerical methods and leverage the 
```

Comment:
I'm not sure about "large-scale" in "large-scale geophysical fluid systems"...
Sure when we write these modules we are thinking of large-scale processes but, e.g., the `TwoDNavierStokes` module can be used to model fluid on a soap bubble... :)

Reply:
Very true, 2D dynamics/turbulence is a broad field. I wasn't sure how to link that in since the module's name is implies geophysics only. I used large-scale to distinguish from non-hydrostatic 3D ocean and atmosphere flows, but you should remove it if you like (I think I also said large-scale later)

Reply:
Yes, but later it was referring to QG modules so it's relevant there :)
OK, I'll remove this and merge. :)